### ISSUE TITLE
Crash in sample app when press on "Echo" button

### ISSUE THREAD

**Issue by chapayGhub** [XWebView/XWebView#89]
```
Main Thread Checker: UI API called on a background thread: 
-[WKWebView evaluateJavaScript:completionHandler:]
```

**Crash stacktrace**
<img width="1907" alt="screen shot 2017-09-21 at 20 56 16" src="https://user-images.githubusercontent.com/878464/30710852-86d60a7e-9f0f-11e7-80ea-247bf82cef53.png">

**New Xcode9 Warning** :)

```
XWVObject.swift: runtime: UI API called from background thread:
WKWebView.evaluateJavaScript(_:completionHandler:) must be used from main thread only
```

<img width="1222" alt="screen shot 2017-09-21 at 21 11 10" src="https://user-images.githubusercontent.com/878464/30711472-7d9ae6e4-9f11-11e7-947c-5d592d8799b6.png">

PS
Xcode 9
iOs 11

**Comment by SalvatoreAD** [comment]
I have the same issue

Use this code for every evaluateJavaScript function
```
DispatchQueue.main.async {
webView.evaluateJavaScript(script, completionHandler: nil)
}
```

and remove Main Thread Checker 
<img width="1512" alt="bibgn" src="https://user-images.githubusercontent.com/15673743/30879785-c037f760-a300-11e7-9dc4-f7da4f9552a6.png">

**Comment by xwv** [comment]
The issue had fixed in #91.
A new method named asyncEvaluateJavaScript is added to WKWebView.
